assignments/life_expectancy/tests/test_cleaning_data_new.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+from life_expectancy.data_cleaning import unpivot, convert_date_format, filter_region
+
+def test_unipivot(life_expectancy_unpivot, life_expectancy_original):
+    response_df = unpivot(life_expectancy_original)
+    # in order to force json file columns to be str
+    life_expectancy_unpivot = life_expectancy_unpivot.astype({"year": "str"})
+    life_expectancy_unpivot = life_expectancy_unpivot.astype({"value": "str"})
+    response_df = response_df.astype({"value": "str"})
+
+    pd.testing.assert_frame_equal(response_df, life_expectancy_unpivot)
+
+
+def test_convert_date_format(
```

Comment:
Having convert_date as a private function saves you of having to test it

Reply:
ok